lib/logger.js

```diff
@@ -134,11 +136,15 @@ class Logger extends EventEmitter {
       this[kLogLevels] = custom_levels
     }
 
+    if (has(options, 'validateLogLevels')) {
+      this[kValidateLogLevels] = Boolean(options.validateLogLevels)
+    }
```

Comment:
Turning validation off makes custom levels worthless.  If using `custom_levels` and `!this[kValidateLogLevels]`, then throw an error stating that defining custom log levels is unnecessary when turning off level validation.

The README should also mention this tidbit, IMO.  Maybe even go so far as to say that this is more of an internal feature and we encourage the use of customLogLevels for end users (kind of like the `agentLog` stuff)?